components/common/map/LocationSearch.tsx

```
@@ -0,0 +1,185 @@
+      console.error('Error searching location:', error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  // Debounced search function
+  const debouncedSearch = useCallback(
+    debounce((query: string) => {
+      searchLocation(query);
+    }, debounceDelay),
+    [debounceDelay]
```

ESLint

[error] Insert `,`

(prettier/prettier)